sonar-scanner-ant/ant-coverage/build.xml

```diff
@@ -94,7 +94,8 @@
 	<target name="sonar" depends="compile">
 		<taskdef uri="antlib:org.sonar.ant" resource="org/sonar/ant/antlib.xml">
 			<!-- Update the following line, or put the "sonar-ant-task-*.jar" file in your "$HOME/.ant/lib" folder -->
-			<classpath path="path/to/sonar/ant/task/lib/sonarqube-ant-task-*.jar" />
+			<!-- <classpath path="path/to/sonar/ant/task/lib/sonarqube-ant-task-*.jar" /> -->
+			<classpath path="/Users/jting/Documents/sonarqube-ant-task-2.7.1.1951.jar" />
```

Comment:
Maybe it is better to leave a generic path here

Reply:
Agree! Updated.